cron/alerts/main.go

```diff
@@ -180,21 +83,21 @@ func getCFRecordsWithSubstring(substring, zoneName string, recs []cloudflare.DNS
 	}
 }
 
-func getCFRecords(config AlertsConfig) map[string][]string {
-	api, err := cloudflare.NewWithAPIToken(config.CFApiToken)
+func getCFRecords(config Config, alertConfig AlertsConfig) map[string][]string {
```

Comment:
Not that it changes much, but could change it to `func (c Config) getCFRecords(...` as passing in multiple "configs" smells funny.

It works, but not entirely sure I like it, or have an idea on how to handle it better yet.

Possible consideration:
- split Config up, so alerts slice is separate from base config, then each alert config has a pointer to base config or extends it some other way.

Reply:
Yes, I agree. Haven't thought of a workable solution yet. The main driver is that the data comes in from AppConfig as one json document, but the list of AlertConfig objects is iterated at the top level of the call structure. I had considered passing an index into the list of AlertConfig objects instead of a copy of the current one, but wasn't crazy about that idea. Your idea might work. I'll experiment with it a bit.